packages/shorebird_cli/lib/src/shorebird_artifacts.dart

```diff
@@ -18,6 +18,9 @@ enum ShorebirdArtifact {
 
   /// The gen_snapshot executable.
   genSnapshot,
+
+  /// TODO
```

Comment:
```suggestion
  /// The updater tools executable
```

Or something like that

Reply:
Done